src/socket/controller/ingame.ts

```diff
@@ -41,6 +42,10 @@ function handleMove(
     if (data.skill) {
         character.isSkillInput = true
     }
+    if (character.items[0] === ItemType.THUNDER) {
+        gameMap.handleItemUse(character)
```

Comment:
* 결국 thunderEffect만 적용시키는건데, 이름이 `applyThunderEffect` 로 바뀌어야할거 같아요 
* 그리고, map 자체에서 컨트롤해야해서 여기서 적용한것 같은데, 

```suggestion
       // objects/player.ts
        useItem(gameMap: CommonMap) {
            switch(){
              // 생략
              
              case ItemType.THUNDER:
                    gameMap.applyThunderEffect(this) // 이렇게 사용할수 있을거 같아요
  
            }      
        }
        
        
```

Reply:
이렇게 하면 ingame.ts에 있는것도 하나로 통합할수있겠네 땡큐